osxphotos/utils.py

```diff
@@ -263,11 +279,13 @@ def list_photo_libraries():
     return lib_list
 
 
-def normalize_fs_path(path: str) -> str:
```

Comment:
Changing unicode normalization from NFD to NFC will likely cause some file names to change which will corrupt existing exports using `--update`.  The switch should be gated to continue using NFD on macos. This was a deliberate choice as this is the default on HFS+ which is still common in the Mac ecosystem.

Reply:
I've changed it back to NFD for macOS. Sadly a lot of software behaves counter-intuitively when the Unicode form coming "from the keyboard" doesn't match the file, so I'm keeping it NFC on Linux/Windows/...